nmrfx-processor-gui/src/main/java/org/nmrfx/processor/gui/log/LogDetailsView.java

```diff
@@ -0,0 +1,92 @@
+package org.nmrfx.processor.gui.log;
+
+import de.jensd.fx.glyphs.GlyphsDude;
+import de.jensd.fx.glyphs.fontawesome.FontAwesomeIcon;
+import javafx.event.ActionEvent;
+import javafx.geometry.HPos;
+import javafx.geometry.Insets;
+import javafx.scene.control.Button;
+import javafx.scene.control.TextArea;
+import javafx.scene.control.TextField;
+import javafx.scene.layout.ColumnConstraints;
+import javafx.scene.layout.GridPane;
+import javafx.scene.layout.Priority;
+import javafx.scene.text.Text;
+import org.nmrfx.chemistry.utilities.NvUtil;
+
+import java.awt.*;
+import java.awt.datatransfer.StringSelection;
+
+
+/**
+ * Class to display the details of a LogRecord.
+ */
+public class LogDetailsView extends GridPane {
+
+    private static final int SPACING = 5;
+    private final TextField datetime = new TextField();
+    private final TextField location = new TextField();
+    private final TextArea message = new TextArea();
+    private final Button copyButton = GlyphsDude.createIconButton(FontAwesomeIcon.COPY);
+    private String level;
```

Comment:
Why using the level as string? It looks like it's only usage is to copy to clipboard, so we could store the LogLevel or even the LogRecord directly here, and convert to string only when copying to clipboard.